src/Common/src/Interop/Unix/System.Private.CoreLib.Native/Interop.ErrNo.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Runtime.InteropServices;
+internal static partial class Interop
+{
+    internal unsafe partial class Sys
+    {
+        [DllImport(Interop.Libraries.CoreLibNative, EntryPoint = "CoreLibNative_GetLastErrNo")]
```

Comment:
We may want to make these RuntimeImports to avoid the PInvoke transitions for these (reduces code size / match CoreCLR perf profile).

Same for Windows SetLastError/GetLastError.